.github/workflows/ci.yml

```diff
@@ -146,13 +146,7 @@ jobs:
       fail-fast: false
       matrix:
         device:
-          - { name: 'trezor-1', archive: 'trezor-firmware',  paths: 'test/work/trezor-firmware' }
-          - { name: 'trezor-t', archive: 'trezor-firmware',  paths: 'test/work/trezor-firmware' }
-          - { name: 'coldcard', archive: 'coldcard-mpy',     paths: 'test/work/firmware/external/micropython/ports/unix/coldcard-mpy test/work/firmware/unix/coldcard-mpy test/work/firmware/unix/l-mpy test/work/firmware/unix/l-port' }
-          - { name: 'bitbox01', archive: 'mcu',              paths: 'test/work/mcu' }
-          - { name: 'jade',     archive: 'jade',             paths: 'test/work/jade/simulator' }
-          - { name: 'ledger',   archive: 'speculos',         paths: 'test/work/speculos' }
-          - { name: 'keepkey',  archive: 'keepkey-firmware', paths: 'test/work/keepkey-firmware/bin' }
+          - { name: 'bitbox02', archive: 'bitbox02',  paths: 'test/bitbox02_simulator' }
```

Comment:
To forward the outputs of `sim-builder` job to device test jobs, the built binaries of simulator are archived with `tar` and uploaded by `actions/upload-artifact@v4`. `archive` name you ask refers to the name of this `.tar.gz` file. Later, it will be used to unzip with `actions/download-artifact@v4` and `tar` by device test jobs.

`test/work` is not an existing folder in the repo. It is being created on the fly. However we upload our simulator statically, before anything is built. Therefore we can not store it in `work` folder, but I can write a copy command in `sim-builder` job to copy in `work` folder and make it look like others here(`test/work/bitbox02_simulator`).

https://github.com/bitcoin-core/HWI/blob/d774d65255e1e78ea4d97846c2b4d26a23febcfb/test/README.md?plain=1#L37 You can also see here that you should create `work` folder if you're building yourself.